starter-code/main.js

```diff
@@ -0,0 +1,229 @@
+					  }
+					})
+						break;
+					case "3":
+					db.collection('companies').find({"founded_year": 2004}).toArray((error, result) => {
+					  if (error) {
+					    console.log(error);
+					    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+					  } else {
+					    console.log(result.length);
+					    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+					  }
+					})
+						break;
+					case "4":
+					db.collection('companies').find({
+						$and: [
+							{"founded_year": 2004},
+							{"founded_month": 2},
+						]}, {name: 1, _id: 0}).toArray((error, result) => {
+					  if (error) {
+					    console.log(error);
+					    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+					  } else {
+					    console.log(result);
+					    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+					  }
+					})
+						break;
+					case "5":
+					db.collection('companies').find({
+						$and: [
+							{"founded_year": 2004},
+							{"founded_month": {$gt: 3}},
+							{"founded_month": {$lt: 7}}
+						]}, {name: 1, _id: 0}).sort({"founded_month": 1}).toArray((error, result) => {
+					  if (error) {
+					    console.log(error);
+					    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+					  } else {
+					    console.log(result);
+					    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+					  }
+					})
+						break;
+					case "6":
+					db.collection('companies').find({"offices.city": "Barcelona"}, {name: 1, _id: 0}).toArray((error, result) => {
+					  if (error) {
+					    console.log(error);
+					    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+					  } else {
+					    console.log(result);
+					    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+					  }
+					})
+						break;
+						case "7":
+						db.collection('companies').find({}, {name: 1, number_of_employees: 1, _id:0}).limit(10).sort({ "number_of_employees": -1}).toArray((error, result) => {
+						  if (error) {
+						    console.log(error);
+						    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+						  } else {
+						    console.log(result);
+						    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+						  }
+						})
+							break;
+					case "8":
+					db.collection('companies').find({"name": "Facebook"}).toArray((error, result) => {
+					  if (error) {
+					    console.log(error);
+					    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+					  } else {
+					    console.log(result);
+					    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+					  }
+					})
+						break;
+					case "9":
+					db.collection('companies').find({"name": "Facebook"}, {number_of_employees: 1, _id: 0}).toArray((error, result) => {
```

Comment:
👍 you could have put the name of the employees in the query as well 
**db.collection('companies').find({name: "Facebook"}, {name: 1, number_of_employees: 1}).toArray((error, result)=>{**